dmoj/executors/base_executor.py

```diff
@@ -180,18 +182,18 @@ def get_runtime_versions(cls):
         return version_cache[key]
 
     @classmethod
-    def parse_version(cls, command, output):
+    def parse_version(cls, command: str, output: str) -> Optional[Iterable[int]]:
         match = cls.version_regex.match(output)
         if match:
             return map(int, match.group(1).split('.'))
         return None
 
     @classmethod
-    def get_version_flags(cls, command):
+    def get_version_flags(cls, command: str) -> List[str]:
         return ['--version']
 
     @classmethod
-    def find_command_from_list(cls, files):
+    def find_command_from_list(cls, files: str) -> Optional[str]:
```

Comment:
`mypy` will not like the implicit `return None` at the end of the function, so it needs to be added.